doc/manuals/admin/logs.md

```diff
@@ -26,6 +26,9 @@ The `-logLevel` option allows to choose which error messages are printed in the
 When Orion runs in foreground (i.e. with the `-fg` [CLI argument](cli.md)), it also prints the same log traces
 (but in a simplified way) on the standard output.
 
+The log level can be changed in run-time, using the admin API exposed by orion.
+For more information, see [management_api](management_api.md).
```

Comment:
Or maybe even better:

"The log level can be changed in run-time, using the [admin API](management_api.md) exposed by Orion.